docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx

```diff
@@ -97,7 +97,7 @@ Begin by writing:
 
 This just adds the basic setup for our smart contract — see [Tutorial 01 Hello World](hello-world) for more on this.
 
-Now, we will add our init method. This is intended to run once to set up the initial state on the zkApp account.
+Now, we will add our initState method. This is intended to run once to set up the initial state on the zkApp account.
```

Comment:
thanks for making this  change, however, the rest of the sentence here became wrong as well: `initState()` isn't designed to be run just once, anyone could run it any number of times. This should be fixed as well IMO (not by you, obviously)

Reply:
noted. will close the PR as it needs other fixes. 🙏🏻